### PR TITLE
Review updates v3

### PR DESCRIPTION
**Pull request by ddgrp**
- back to top link
- minor styling changes

### CODE REVIEW

File: _includes/html/header [hunk 1]
```diff
@@ -4,6 +4,7 @@
 {% assign headerimage = include.headerimage %}
 {% endif %}
 <header id="top">
+    <a name="backtotop"></A>
```
Comment (mringer) [inline _includes/html/header]: there is already a id="top" you can link to just above the <a> you added, we don't need a new one just link to the top of the header please.

File: _includes/html/backtotop [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<p>
+<br/>
+<a id="backtoptop" href="#backtotop" class="fa fa-chevron-circle-up"> Back to top</a>
```
Comment (mringer) [inline _includes/html/backtotop]: Just link to the existing top id in the header

**Review by mringer — CHANGES_REQUESTED** [review]
there is already a top id in the header, we don't need two elements to accomplish the same task.

**Comment by ddgrp** [comment]
@matthewringer the update of re-using the existing "top" id element is implemented along with minor css change.

**Pull request merged**